azure-pipelines.yml

```diff
@@ -254,3 +254,23 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: RichCodeNav_Indexing
+  pool:
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2017.Open
+  variables:
+    EnableRichCodeNavigation: true
+  timeoutInMinutes: 200
+  condition: and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'master'))
```

Comment:
See earlier comment: I suspect this needs to be it's own .yml and so you can configure that directly.